portfolio/src/main/webapp/index.html

```diff
@@ -37,6 +37,11 @@ <h3 id="Experience">Experience</h3>
      <h2 id="Contact">Contact Me</h2> 
      <p style="color:purple;">Bpaynes4291@gmail.com</p> 
      <a href="https://github.com/BlessP/Step"><p style="color:purple;">Github</p></a>
+     <div class="buttons" id="message">
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Click for my name!</button>
```

Comment:
A button needs to have a type (see the [button tag doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button))

In this case, you could use `type="button"`?